compute_tools/src/bin/fast_import.rs

```diff
@@ -677,11 +678,35 @@ pub(crate) async fn main() -> anyhow::Result<()> {
                 spec,
                 source_connection_string,
                 destination_connection_string,
-                args.working_directory,
+                args.working_directory.clone(),
                 args.pg_bin_dir,
                 args.pg_lib_dir,
             )
-            .await?;
+            .await
+        }
+    };
+
+    if let Some(s3_prefix) = args.s3_prefix {
+        info!("write job status to s3");
+        {
+            let status_dir = args.working_directory.join("status");
+            if std::fs::exists(&status_dir)?.not() {
+                std::fs::create_dir(&status_dir).context("create status directory")?;
+            }
+            let status_file = status_dir.join("fast_import");
```

Comment:
Not sure about name of the file. Need opinion on that

Reply:
I would use the same name as in the fast import (`Command::PgData`) flow: `$root/status/pgdata/status`. Or we can change it for both.

Reply:
I am not sure I want to break compatibility now without a specific need. I think current thing are ok, I can put command name and args into the file as well

Reply:
I don't mind that much. We wouldn't break anything by changing the ps import flow since that's not in prod, but up to you.